src/objective-c/GRPCClient/GRPCCall.h

```diff
@@ -155,6 +155,18 @@ typedef NS_ENUM(NSUInteger, GRPCErrorCode) {
 };
 
 /**
+ * Flags for options of a gRPC call
```

Comment:
Agreed with the principle. My understanding was they meant just exactly the same in this case, though?